airflow/config_templates/config.yml

```
@@ -361,6 +361,15 @@
       type: string
       example: ~
       default: "downstream"
+    - name: default_execution_timeout
```

I think it should be 
```suggestion
    - name: default_task_execution_timeout
```

like: `default_task_retries`, `default_task_weight_rule`, etc...

---

Thank you, you're quite right.

---

renamed all the configuration parameter and variable names accordingly ✅